src/cookiebanner.js

```diff
@@ -483,11 +483,11 @@ THE SOFTWARE.
               });
             }
 		
-	    // Agree and close banner after N milliseconds
-	    if (this.options.acceptOnTimeout) {
-	      // Validate this.options.acceptOnTimeout as numeric
-	      if(!isNaN(parseFloat(this.options.acceptOnTimeout)) && isFinite(this.options.acceptOnTimeout)) {
-	        setTimeout(function() { self.agree_and_close(); }, this.options.acceptOnTimeout);
+           // Agree and close banner after N milliseconds
```

Comment:
This entire block still seems to be indented 1 position left to what it should be? (see the closing brace in line 484)